dspace-api/src/main/java/org/dspace/app/mediafilter/ImageMagickThumbnailFilter.java

```diff
@@ -132,12 +137,19 @@ public static File getImageFile(File f, int page) throws IOException, Interrupte
     	f2.deleteOnExit();
     	ConvertCmd cmd = new ConvertCmd();
 		IMOperation op = new IMOperation();
+		Info imageInfo = new Info(f.getAbsolutePath(),true);
 		String s = "[" + page + "]";
 		op.addImage(f.getAbsolutePath()+s);
                 if (flatten)
                 {
                     op.flatten();
                 }
+		String imageClass = imageInfo.getImageClass();
+		// PDFs using the CMYK color system can be handled specially if profiles are defined
+		if (imageClass.contains("CMYK") && cmyk_profile != null && srgb_profile != null) {
+			op.profile(cmyk_profile);
+			op.profile(srgb_profile);
```

Comment:
I suppose I could add some more conditional logic to test if there was an embedded profile, though I'm not sure how common PDFs with embedded profiles would be. Anecdotally, on our repository I re-processed all our PDFs bitstreams and noted that ~3,000 items out of 15,000 were using the CMYK color system, which is a pretty big win if you ask me!

I will try to find (or create) some PDFs with embedded profiles to do some experiments with this this week.